src/Etterna/Models/NoteData/NoteDataUtil.cpp

```diff
@@ -172,7 +172,6 @@ LoadFromSMNoteDataStringWithPlayer(NoteData& out,
 						int iHeadRow;
 						if (!out.IsHoldNoteAtRow(iTrack, iIndex, &iHeadRow)) {
 							int n = intptr_t(endLine) - intptr_t(beginLine);
-							LOG->Warn("Unmatched 3 in \"%.*s\"", n, beginLine);
```

Comment:
No changed log message here?

Reply:
I did change it, but then mina removed it.

Reply:
Removed in `2533bf8`.